### ISSUE TITLE
New GitHub tokens marked invalid

### ISSUE THREAD

**Issue by paulgrow-octane** [jasperapp/jasper#205]
In response to an automated message from Github API regarding an [outdated token format](https://github.blog/2021-04-05-behind-githubs-new-authentication-token-formats/), I generated a new access token to use with Jasper. Unfortunately, Jasper reports that the token is not valid:

![image](https://user-images.githubusercontent.com/74255970/113723076-9bfa0480-96b6-11eb-9366-a51b41ba5eb2.png)

Now I cannot get the app working at all 😭

**Comment by hiroyuki-sato** [comment]
Hello, @paulgrow-octane 

I've faced the same issue. Jasper 1.0.2 on macOS.
 It is my workaround that edits `~/Library/Containers/io.jasperapp/data/Library/Application Support/jasper/io.jasperapp/config.json` directory.

```json
    "github": {
      "accessToken": "github token here",
```

**Comment by paulgrow-octane** [comment]
@hiroyuki-sato thanks a lot for your recommendation. I actually needed to migrate to a new laptop anyways, and got stuck on the #60 issue. The only way I managed to work around it was to copy `config.json` and `main.db` into the same directory on the new machine (and, of course, update the token) and everything is working again.

A bit concerning that #60 is still open though... had it not been for the existing configuration files, I would have been completely stuck 😞
